tests/functional_tests/ffront_tests/test_unrollPowOp.py

```diff
@@ -0,0 +1,160 @@
+import pathlib
+
+from eve.pattern_matching import ObjectPattern
+from functional.ffront import common_types as ct, field_operator_ast as foast
+from functional.ffront.func_to_foast import UnrollPowerOp
+
+
+def test_power_0():
+    def func():
+      return a**0
+    testee = FieldOperatorParser.apply_to_function(func)
```

Comment:
This is an interesting strategy and I think would be nice if we could use it everywhere. We should probably relax the typing requirement for testing @tehrengruber.
But for now I would use the same structure everywhere. It distracts from the important part (the reader might try to figure out why this test is written that way and all other tests in this other way).